RxEureka/Classes/RxEureka.swift

```diff
@@ -15,12 +15,17 @@ extension BaseRow: ReactiveCompatible { }
 public extension Reactive where Base: BaseRow, Base: RowType {
 
   public var value: ControlProperty<Base.Cell.Value?> {
+    var base: Base? = self.base
```

Comment:
What do you think about adding a weak reference to self ([weak self]) in the block instead of a strong reference to the base in the value accessor scope ? This should fix the leak too (not tested).

Reply:
This is what I wanted to do at the first attempt. Does not work because the compiler complains. Maybe this works, but I did not test it - and not sure if we want to add the class constraint here: https://stackoverflow.com/questions/24066304/how-can-i-make-a-weak-protocol-reference-in-pure-swift-without-objc

Reply:
Gave it another thought. Your right. Using weak is prettier & better. And since BaseRow is always a class anyways we can safely apply this constraint to the extension.

Is it possible for you to make these changes?

Reply:
I reproduced the issue and fixed the leak on #14. Feel free to give me a feedback on the leak and the code.